packages/@ama-sdk/schematics/project.json

```diff
@@ -37,7 +37,25 @@
       "executor": "nx:run-script",
       "options": {
         "script": "build:builders"
-      }
+      },
+      "outputs": [
```

Comment:
What is the difference with the default one from `nx.json`?
If there is one, can not it be updated to the `nx.json` list?

Reply:
We add the dist/src which contains the schematics interface, I don't think this is something we want on non schematics projects.